packages/web-components/src/components/va-button-pair/va-button-pair.tsx

```diff
@@ -0,0 +1,144 @@
+import { Component, Event, EventEmitter, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'va-button-pair',
+  styleUrl: 'va-button-pair.css',
+  shadow: true,
+})
+export class VaButtonPair {
+  /**
+   * If `true`, button pair will use Confirm and Cancel for button text.
```

Comment:
Can we add something here describing the Yes/No text for when `confirm` and `continue` are both false? I don't want that to be unexpected.

Reply:
In storybook I see that the default story has the Yes/No text, so this isn't as unexpected as I thought :upside_down_face: